MaveDB.pm

```diff
@@ -150,7 +150,7 @@ sub new {
   $self->add_file($file);
 
   # Parse column names
-  my $cols = $param_hash->{cols} || "urn:score:nt:pro";
+  my $cols = $param_hash->{cols} || "urn:score:nt:hgvs";
```

Comment:
```suggestion
  my $cols = $param_hash->{cols} || "urn:score:nt:pro";
```

It would be good to keep the default col as `pro` instead of `hgvs` here. The reason is 2-fold - 
- the `nt` and `pro` both are coming from the scores file providing HGVSg and HGVSp respectively. If we are giving the HGVSg from scores file it make sense to give the HGVSp from the scores file as well.
- also `hgvs` col is the HGVS that the MAVE results are match against the Variant recorder when creating the data files. And it is not always HGVSp - it can be HGVSg as well. So providing HGVSg twice using `nt` and `hgvs` would not be necessary. Other than matching AA with VEP results we might as well hide it by default as before.